ci/recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - python 3.5*
     - setuptools
     - pandas
```

Comment:
should pandas be pinned to 0.23.0 or is this change backwards-compatible?

Reply:
Should be backwards compatible - I tested pandad 0.22 and 0.23.

Reply:
👍